spec/helpers/image_helper_spec.rb

```diff
@@ -0,0 +1,129 @@
+require 'spec_helper'
```

Comment:
This test replicates the code in line 36 of image_helper.rb, but does not actually call the image helper code.  It would be preferable to refactor image_helper to move the page size calculation to its own method, then call it from this test.